client/src/components/common/boost/BoostConfigList.tsx

```diff
@@ -12,6 +12,13 @@ export interface BoostConfigListProps {
   onDeleteConfig: (configName: string) => void;
 }
 
+const fileNameSuffixes = [
+  '_rider.json',
+  '_bike.json',
+  '_track.json',
+  '_powerPlan.json',
+];
+
```

Comment:
This somewhat duplicates the lookup object in `api/v3/boost.ts` that matches config type to suffix. It would be nice to enforce that with a shared object or type.

Another idea would be to introduce `addConfigSuffix: (name: string, config: ConfigT) => string` and `removeConfigSuffix: (configName: string) => string` as util functions that encode/decode the config file name as currently it is just `_configType.json`. 
This would make the encoding/decoding process much easier (line79 in `api/v3/boost` and below)

Reply:
Really great insight/idea, I was able to build upon it to actually simplify both into just one function.